planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -927,6 +939,10 @@ ObstacleAvoidancePlanner::generateOptimizedTrajectory(
   // calculate trajectory with EB and MPT
   auto optimal_trajs = optimizeTrajectory(path, cv_maps);
 
+  // calculate velocity
+  // NOTE: Velocity is not considered in optimization.
+  calcVelocity(path.points, optimal_trajs.model_predictive_trajectory);
```

Comment:
just memo

If optimization fails, the previously optimized trajectory is used. When this optimization failure occurs while the stop line changes from red to green, the velocity in the previous trajectory is not updated from red to green, and the ego keeps stopping.
Therefore calcVelocity was moved after optimization to update velocity even if the optimization fails.